README.md

```diff
@@ -72,11 +77,15 @@ To change configuration values, follow the below steps:
 
 ### Installation
 
-Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of the eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
+Using FTP transfer the following files to the /usr/ directory of the Ewon via FTP. 
+* ConnectorConfig.json, an example file is provided in the starting-files directory of the release zip.
+* flexy-osisoft-connector-x-x-x.jar, the latest version can be downloaded from the github releases.
+* JVMRUN, this  file is provided in the starting-files directory of the release zip.  
```

Comment:
Replace `JVMRUN` with `jvmrun` which is the proper capitalization of the actual file. To make it stand out, you can make it italics or putting it in a code block.